### PR TITLE
Person info card

### PR DESCRIPTION
**Pull request by c0m1t**
Create person-info-card component
Implement it in peer-review-page

### CODE REVIEW

File: src/pages/peer-review-page/PersonInfoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import { i18n } from '@lingui/core';
+import { Avatar, Button, Card, CardContent, CardHeader, Divider, makeStyles, Theme } from '@material-ui/core';
+import { red } from '@material-ui/core/colors';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import graphql from 'babel-plugin-relay/macro';
+import React, { useCallback } from 'react';
+import { useFragment } from 'react-relay/hooks';
+import { LanguageCodes } from 'src/core/locales/types';
+import { useFinalSubmitMutation } from 'src/pages/peer-review-page/finalSubmit.mutation';
+import { PersonInfoCard_user$key } from 'src/pages/peer-review-page/__generated__/PersonInfoCard_user.graphql';
```
Comment (m-davoodi) [inline src/pages/peer-review-page/PersonInfoCard.tsx]:
```javascript
import { useFinalSubmitMutation } from './finalSubmit.mutation';
import { PersonInfoCard_user$key } from './__generated__/PersonInfoCard_user.graphql';
```

File: src/pages/peer-review-page/PersonInfoCard.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  const finalSubmitMutation = useFinalSubmitMutation();
+
+  const handleSubmit = useCallback(() => {
+    finalSubmitMutation({ input: { revieweeId: user.id, state: 'DONE' } });
+  }, [user, finalSubmitMutation]);
+
+  const numberOfProjects = localizeNumber(user.projectReviews.length, i18n.language as LanguageCodes);
+
+  const disabled = !user.projectReviews.every(projectReview =>
+    projectReview.comments.map(comment => (!!comment ? !!comment.rating && !!comment.rating : false)),
```
Comment (m-davoodi) [inline src/pages/peer-review-page/PersonInfoCard.tsx]: why?
```js
!!comment.rating && !!comment.rating
```

File: src/locales/fa/messages.po [hunk 3]
```diff
@@ -151,6 +152,10 @@ msgstr "باریکلا! چه خوب که همه رو داری انجام مید
 msgid "He/She Needs Improvement"
 msgstr "کمتر از حد انتظاری که ازش میره"
 
+#: src/pages/peer-review-page/PersonInfoCard.tsx:18
+msgid "He/She asked your review on {numberOfProjects} project(s)"
+msgstr "در {numberOfProjects} پروژه از شما نظر خواسته"
```
Comment (sijav) [inline src/locales/fa/messages.po]:
```suggestion
msgstr "در {numberOfProjects} پروژه از شما نظر خواسته شده است"
```

File: src/pages/peer-review-page/PeerReviewPage.tsx [hunk 4]
```diff
@@ -1,9 +1,13 @@
 import { i18n } from '@lingui/core';
 import { Box, Container, makeStyles, Paper, Tabs, Theme } from '@material-ui/core';
 import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import graphql from 'babel-plugin-relay/macro';
 import React, { Suspense } from 'react';
+import { useLazyLoadQuery } from 'react-relay/hooks';
 import { Redirect, Route, Switch, useParams } from 'react-router-dom';
 import CriteriaPage from 'src/pages/criteria-page/CriteriaPage';
+import { PersonInfoCard } from 'src/pages/peer-review-page/PersonInfoCard';
+import { PeerReviewPageQuery } from 'src/pages/peer-review-page/__generated__/PeerReviewPageQuery.graphql';
```
Comment (sijav) [inline src/pages/peer-review-page/PeerReviewPage.tsx]:
```suggestion
import { PeerReviewPageQuery } from './__generated__/PeerReviewPageQuery.graphql';
```

File: src/pages/peer-review-page/PeerReviewPage.tsx [hunk 5]
```diff
@@ -1,9 +1,13 @@
 import { i18n } from '@lingui/core';
 import { Box, Container, makeStyles, Paper, Tabs, Theme } from '@material-ui/core';
 import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import graphql from 'babel-plugin-relay/macro';
 import React, { Suspense } from 'react';
+import { useLazyLoadQuery } from 'react-relay/hooks';
 import { Redirect, Route, Switch, useParams } from 'react-router-dom';
 import CriteriaPage from 'src/pages/criteria-page/CriteriaPage';
+import { PersonInfoCard } from 'src/pages/peer-review-page/PersonInfoCard';
```
Comment (sijav) [inline src/pages/peer-review-page/PeerReviewPage.tsx]:
```suggestion
import { PersonInfoCard } from './PersonInfoCard';
```

File: src/pages/peer-review-page/PersonInfoCard.tsx (hunk 2 above)
Comment (sijav) [inline src/pages/peer-review-page/PersonInfoCard.tsx]:
```suggestion
    projectReview.comments.map(comment => (comment?.rating || false)),
```

**Pull request merged**